File: src/cpp/lib/lawyer.cpp
#include <iostream> 
#include <string>
using namespace std;

class Lawyer : public Worker
{
    protected:
    string law_firm;

    public:
    void setdata(string fname, string lname, string g, int a, float h, float s, float weekhours, string lfirm)
    {
        first_name = fname;
        last_name = lname;
        gender = g;
        age = a;
        height = h;
        salary = s;
        weekly_hours = weekhours;
        law_firm = lfirm;
    }
    void displaydata()
    {
        cout<<law_firm<<endl;
    }
    void talk()
    {
        cout<<"Hello! I am "<<first_name<<" "<<last_name<<", I am lawyer that works "<<weekly_hours
        <<" a week at "<<law_firm<<" and I have a salary of "<<salary<<"."<<endl;
    }
};

File: src/python/lib/lawyer.py
from lib.worker import Worker
from dataclasses import dataclass

@dataclass
class Lawyer(Worker):


    law_firm: str

    

    def talk(self):


        print("Hello! I am {} {}, I am a lawyer that works {} a week at {} and I have a salary of {}.".format(self.first_name, self.last_name, self.weekly_hours, self.law_firm, self.salary))